File: app/src/main/java/com/mbestavros/geometricweather/daily/adapter/model/DailyWind.java
package com.mbestavros.geometricweather.daily.adapter.model;

import com.mbestavros.geometricweather.basic.model.weather.Wind;
import com.mbestavros.geometricweather.daily.adapter.DailyWeatherAdapter;

public class DailyWind implements DailyWeatherAdapter.ViewModel {

    private Wind wind;

    public DailyWind(Wind wind) {
        this.wind = wind;
    }

    public Wind getWind() {
        return wind;
    }

    public void setWind(Wind wind) {
        this.wind = wind;
    }

    public static boolean isCode(int code) {
        return code == 4;
    }

    @Override
    public int getCode() {
        return 4;
    }
}

File: app/src/main/java/org/breezyweather/daily/adapter/model/DailyWind.kt
package org.breezyweather.daily.adapter.model

import org.breezyweather.common.basic.models.weather.Wind
import org.breezyweather.daily.adapter.DailyWeatherAdapter

class DailyWind(
    val wind: Wind
) : DailyWeatherAdapter.ViewModel {
    override val code = 4

    companion object {
        fun isCode(code: Int) = (code == 4)
    }
}
